ui/pages/Public/components/TermsOfService.jsx

```diff
@@ -38,16 +42,16 @@ export default () => (
         authentication credentials to anyone; a Broad employee should never ask you to reveal them.
```

Comment:
Missing text, "Maintain the confidentiality of your authentication credentials such as your password. Do not reveal your"